src/deadline/client/cli/_groups/asset_group.py

```diff
@@ -300,15 +303,65 @@ def asset_diff(root_dir: str, manifest_dir: str, raw: bool, **args):
 
 
 @cli_asset.command(name="download")
-@click.option("--farm-id", help="The AWS Deadline Cloud Farm to use.")
-@click.option("--queue-id", help="The AWS Deadline Cloud Queue to use.")
-@click.option("--job-id", help="The AWS Deadline Cloud Job to get. ")
+@click.option("--farm-id", help="The AWS Deadline Cloud Farm to use. ")
+@click.option("--queue-id", help="The AWS Deadline Cloud Queue to use. ")
+@click.option("--job-id", required=True, help="The AWS Deadline Cloud Job to get. ")
```

Comment:
Yea that should work too, but I was thinking since the command itself is explicitly trying to download the input manifests of a job, it would make more sense to have the job it downloads from to be explicitly specified ?

Reply:
looking at other CLI examples though like the job group commands, doesn't specify job-id for download. Do you think it'll be best to follow those ?

Reply:
If it's a convention for other commands, not following it may lead to customer's confusion.